pkg/storage/tsdb/config.go

```diff
@@ -34,14 +35,34 @@ type Config struct {
 	GCS gcs.Config `yaml:"gcs"`
 }
 
+// RangeList is the block ranges for a tsdb
+type RangeList []time.Duration
+
+// String implements the flag.Var interface
+func (b RangeList) String() string { return "RangeList is the block ranges for a tsdb" }
+
+// Set implements the flag.Var interface
+func (b RangeList) Set(s string) error {
```

Comment:
Would you mind adding unit tests on this config, please?

Reply:
Will do